### DISCUSSION TITLE
Adding IAM Users

### DISCUSSION THREAD

**Discussion by abhijna** [nih-cfde/training-and-engagement#420] [AWS] [answered]
How to make IAM users for AWS lessons?

**Comment by abhijna** [comment]
Please follow instructions here to make an IAM account: 
https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started_create-admin-group.html

**Reply by abhijna** [reply]
You'll have to work through all the tabs under the "getting started" section

**Comment by abhijna** [comment, marked as answer]
We now have a detailed tutorial on how to set up IAM accounts: https://github.com/nih-cfde/training-and-engagement/wiki/IAM-account-for-AWS

**Comment by ctb** [comment]
yay!
